lib/chore/strategies/worker/preforked_worker_strategy.rb

```diff
@@ -23,6 +23,7 @@ def initialize(manager, opts = {})
         @self_read, @self_write = IO.pipe
         trap_signals(NUM_TO_SIGNAL, @self_write)
         @worker_manager = WorkerManager.new(create_master_socket)
+        at_exit { delete_socket_file}
```

Comment:
It seem fine in theory, although I think it obscures shutdown behavior. Has this been tested to work?